### PR TITLE
Fixes Messenger bag not spawning on wizards

### LINKED ISSUE #17083: Setting your bag style to "messenger bag" spawns you with no backpack at all as wizard
#### Steps to reproduce 

- Set "Messenger bags" as your backpack option in your preferences
- roll wizard (gamemode was ragin')
- notice how you have no backpack and whine in ahelps

#### Specific information for locating

https://github.com/vgstation-coders/vgstation13/blob/e5213f1c128032543b6bbd5acf2a6b8c9178cb38/code/game/gamemodes/wizard/wizard.dm#L199-L204

#### Issue bingo

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by ShiftyRail**
Fixes #17083

### CODE REVIEW

File: data/mode.txt [hunk 1]
```diff
@@ -1 +1 @@
-sandbox
+wizard
```
Comment (DamianX) [inline data/mode.txt]: image

Comment (ShiftyRail) [reply data/mode.txt]: Fug

**Comment by ShiftyRail** [comment]
It's not really a hotfix since the bug has probably been here since we ported Messenger bags, I think.

**Comment by DamianX** [comment]
gives me an excuse to speedmerge though

**Pull request merged**